nfc/libs/NfcCoreLib/lib/NciCore/phNciNfc_Core.h

```diff
@@ -14,6 +14,21 @@
 #define PHNCINFC_GETNCICORECONTEXT() gpphNciNfc_CoreContext
 
 /**
+* \ingroup grp_nci_nfc
+* \brief The supported NCI version of the specification
+*/
+#define PH_NCINFC_VERSION_MAJOR_MASK                (0xF0)
+#define PH_NCINFC_VERSION_MINOR_MASK                (0x0F)
+
+#define PH_NCI1xNFC_VERSION_MAJOR                   (0x01)
+#define PH_NCI1xNFC_VERSION_MINOR                   (0x00)
+#define PH_NCI2xNFC_VERSION_MAJOR                   (0x02)
+#define PH_NCI2xNFC_VERSION_MINOR                   (0x00)
+
+#define PH_NCI1xNFC_VERSION                         ((PH_NCI1xNFC_VERSION_MAJOR << 4) | PH_NCI1xNFC_VERSION_MINOR)
+#define PH_NCI2xNFC_VERSION                         ((PH_NCI2xNFC_VERSION_MAJOR << 4) | PH_NCI2xNFC_VERSION_MINOR)
+
+/**
```

Comment:
Actually I needed to move those definitions for phNciNfc_DbgDescription.c for Nci CoreInitRsp display. It has a different format depending of its NCI version. I haven't found a better way.